README.md

```diff
@@ -1,15 +1,228 @@
+```
+
+To execute this workflow on NCI Gadi HPC, you will need to specify the following flags to the default run command:
+
+```
+nextflow run main.nf --input samples.csv --ref /path/to/ref --gadi-account <account> --whoami <username> -profile gadi
+```
+
+Please be aware that as of October 2023, NCI Gadi HPC queues do not have external network access. This means you will not be able to pull the workflow code base or containers if you submit your nextflow run command as a job on any of the standard job queues. NCI currently recommends you run your Nextflow head job either in a GNU screen or tmux session from the login node or submit it as a job to the copyq.
+
+The NCI Gadi config currently runs all tasks on the normal queue. This config uses the `--gadi-account` flag to assign a project code to all task job submissions for billing purposes. The version of Nextflow installed on Gadi has been modified to make it easier to specify resource options for jobs submitted to the cluster. See NCI's [Gadi user guide](https://opus.nci.org.au/display/DAE/Nextflow) for more details.
+
+
+Please use the PBS script `PBS_gadi_runPipeline.sh` present in the main `Somatic-shortV-nf/` directory for testing the pipeline with test datasets. 
+
+
+## Benchmarking
+
+Coming soon!
+
+
+## Workflow summaries
+### Metadata
+
+|metadata field     | Somatic-shortV-nf / v1.0     |
+|-------------------|:--------------------------------- |
+|Version            | 1.0                               |
+|Maturity           | under development                 |
+|Creators           | Tracy Chew, Cali Willet,Nandan Deshpande                    |
+|Source             | NA                                |
+|License            | GNU General Public License v3.0   |
+|Workflow manager   | NextFlow                          |
+|Container          | See component tools               |
+|Install method     | NA                                |
+|GitHub             | https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf                            |
+|bio.tools          | NA                                |
+|BioContainers      | NA                                |
+|bioconda           | NA                                | 
+
+### Component tools
+
+To run this pipeline you must have Nextflow and Singularity installed on your machine. All other tools are run using containers. 
+
+|Tool         | Version  |
+|-------------|:---------|
+|Nextflow     |>=20.07.1 |
+|Singularity  |   3.11.3       |
+|GATK  |   4.4.0.0       |
+|SnpEff       |  5.2   |
 
 ## Additional notes
+Resources
+- [Nextflow documentation](https://www.nextflow.io/docs/latest/index.html)
 
 ## Help / FAQ / Troubleshooting
+* It is essential that the reference genome you're using contains the same chromosomes, contigs, and scaffolds as the BAM files. To confirm what contigs are included in your indexed BAM file, you can use Samtools idxstats:
+```
+samtools idxstats input.bam | cut -f 1
+```
```

Comment:
This is taken directly from GSV. Does this even apply to this workflow?

Reply:
- The input to this pipeline are bam files (generated by other tools)
- The user will be choosing a Reference genome for this workflow.
- I thought it was a good idea for them to confirm that the bams are compatible with the reference-fasta 
  - Let me know if it makes sense or I will remove.

Reply:
See #9